### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by roverflow**
##### SUMMARY
Fixes Unit tests

##### ISSUE TYPE
- Test fixes

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -17,3 +17,7 @@ collections/
 # Tracked but not manually edited
 
 # Tracked but manually formatted
+
+# WIP
+tests/integration/targets/ios_config/templates/config.js
```
Comment (NilashishC) [inline .prettierignore]: This file doesn't exist in this repo.

Comment (roverflow) [reply .prettierignore]: Will remove, thanks

File: tests/unit/plugins/action/test_es_data_inputs_network.py [hunk 2]
```diff
@@ -370,15 +369,49 @@ def test_es_data_inputs_network_merged_idempotent(self, conn, monkeypatch):
...
+        ):
+            return get_response
+
         monkeypatch.setattr(SplunkRequest, "get_by_path", get_by_path)
+        monkeypatch.setattr(SplunkRequest, "create_update", create_update)
 
         # tcp_cooked
-        get_response = RESPONSE_PAYLOAD["tcp_cooked"]
+        get_response = {
+            "entry": [
```
Comment (NilashishC) [inline tests/unit/plugins/action/test_es_data_inputs_network.py]: Instead of rewriting this here, can we not do `RESPONSE_PAYLOAD["tcp_cooked"]["entry"]`?

Comment (roverflow) [reply tests/unit/plugins/action/test_es_data_inputs_network.py]: Will do, thanks

File: changelogs/fragments/module_plugins_bug_fix.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - Fixed argspec validation for plugins with empty attributes.
+  - Fixed tests for all plugins.
```
Comment (NilashishC) [inline changelogs/fragments/module_plugins_bug_fix.yml]: This doesn't need to be a part of changelog.

File: changelogs/fragments/module_plugins_bug_fix.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - Fixed argspec validation for plugins with empty attributes.
```
Comment (NilashishC) [inline changelogs/fragments/module_plugins_bug_fix.yml]:
```suggestion
  - Fixed argspec validation for plugins with empty task attributes when run with Ansible 2.9.
```

Comment (roverflow) [reply changelogs/fragments/module_plugins_bug_fix.yml]: Will do, thanks!

**Review by NilashishC — APPROVED** [review]

**Pull request merged**